File: GulfRuby.Web/Binding/Models/CompanyInformationModel.js
(function (cr) {
    var CompanyInformationModel = function () {

        var self = this;
        self.ID = 0;
        self.TourismLicense = '';
        self.TourismExpiry = '';
        self.Rent = '';
        self.RentExpiry = '';
        self.WasteNumber = '';
        self.WasteNumberExpiry = '';
        self.CivilDefence = '';
        self.CivilDefenceExpiry = '';
        self.Sponsorship = '';
        self.SponsorshipExpiry = '';
        self.AddedTime = '';
        self.LastModifiedBy = '';
    }
    cr.CompanyInformationModel = CompanyInformationModel;
}(window.GulfRuby));

File: GulfRuby.Web/Models/CompanyInformationModel.cs
using System;

namespace GulfRuby.Web.Models
{
    public class CompanyInformationModel
    {

        public byte ID { get; set; }

        public string TourismLicense { get; set; }

        public string TourismExpiry { get; set; }

        public string Rent { get; set; }

        public string RentExpiry { get; set; }

        public string WasteNumber { get; set; }

        public string WasteNumberExpiry { get; set; }

        public string CivilDefence { get; set; }

        public string CivilDefenceExpiry { get; set; }

        public string Sponsorship { get; set; }

        public string SponsorshipExpiry { get; set; }

        public string LastModifiedBy { get; set; }

        public DateTime AddedTime { get; set; }

    }
}